INSTALL.md

```diff
@@ -530,7 +530,7 @@ Setting the FIPS HMAC key
 
 As part of its self-test validation, the FIPS module must verify itself
 by performing a SHA-256 HMAC computation on itself. The default key is
-the SHA256 value of "the holy handgrenade of antioch" and is sufficient
+the SHA256 value of "holy hand grenade of antioch" and is sufficient
```

Comment:
https://github.com/openssl/openssl/blob/f08be096517f9bdae8a9d1d837748237db4d13a9/Configure#L284

Reply:
Looks like this mismatch has been there since it was added 4 years ago :)